infrastructure/run_ansible_vagrant

```diff
@@ -46,16 +46,15 @@ function buildJars() {
 
 function buildMigrations() {
   local targetFolder=./ansible/roles/database/flyway/files/
-  local zipFile=migrations.zip
-  if [ ! -e $targetFolder/$zipFile ]; then
-    if [ ! -e ../lobby-db/build/distributions/$zipFile ]; then
```

Comment:
Note, small simplification where this `if` statement was removed. If the target file DNE then we'll proceed to run gradle. Before we checked as well if the gradle output existed to skip the gradle run, instead now we'll just run gradle anyways.